irohad/api/command_service.cpp

```diff
@@ -15,37 +15,40 @@ limitations under the License.
 */
 
 #include "command_service.hpp"
+#include <grpc++/server_context.h>
 #include <ordering/queue.hpp>
 #include <validation/stateless/validator.hpp>
 
-namespace connection {
-  namespace api {
-
-    using namespace iroha::protocol;
-
-    std::function<void(const Transaction&)> dispatchToOrdering;
-
-    void receive(
-        std::function<void(const iroha::protocol::Transaction&)> const& func) {
-      dispatchToOrdering = func;
-    }
-
-    grpc::Status CommandService::Torii(grpc::ClientContext* context,
-                                       const Transaction& request,
-                                       ToriiResponse* response) {
-      // TODO: Use this to get client's ip and port.
-      (void)context;
-
-      if (validator::stateless::validate(request)) {
-        dispatchToOrdering(request);
-        // TODO: Return tracking log number (hash)
-        *response = ToriiResponse();
-      } else {
-        // TODO: Return validation failed message
-        *response = ToriiResponse();
-      }
-      return grpc::Status::OK;
+namespace api {
+
+  using namespace iroha::protocol;
+
+  std::function<void(const Transaction&)> dispatchToOrdering;
+
+  void receive(
+      std::function<void(const iroha::protocol::Transaction&)> const& func) {
+    dispatchToOrdering = func;
+  }
+
+  grpc::Status CommandService::Torii(grpc::ServerContext* context,
+                                     const Transaction* request,
+                                     ToriiResponse* response) {
+    // TODO: Use this to get client's ip and port.
+    (void)context;
+
+    if (validation::stateless::validate(*request)) {
+      dispatchToOrdering(*request);
+      // TODO: Return tracking log number (hash)
+      *response = ToriiResponse();
+      response->set_code(ResponseCode::OK);
+      response->set_message("successfully dispatching to ordering.");
+    } else {
+      // TODO: Return validation failed message
+      *response = ToriiResponse();
```

Comment:
Is this need ?
If response is null, we should write that
`if( !response ) response = new ToriiResponse();`

line:42 : is same.